doc/administrator/galaxy/pipelines/mentalist/index.md

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
Could you link this file in https://github.com/phac-nml/irida/blob/development/doc/administrator/galaxy/existing-galaxy/index.md#manual-installation-of-tools and https://github.com/phac-nml/irida/blob/development/doc/administrator/galaxy/setup/index.md#manual-installation-of-tools similar to the other tools?

Reply:
Great.  Thanks.